clientSide/src/components/EditBooking/EditBooking.jsx

```diff
@@ -0,0 +1,82 @@
+import { Link, useNavigate, useParams } from "react-router-dom";
+import "./EditBooking.css";
+import GrayContainer from "../UiComponents/GrayContainer";
+import GrayColumn from "../UiComponents/GrayColumn";
+import LabeledInput from "../LabeledInput/LabeledInput";
+import { useState } from "react";
+import { onChangeHandler } from "../../functions/onChangeHandlers";
+import DropDown from "../DropDown/DropDown";
+import Button from "../Button/Button";
+import NavBar from "../NavBar/Navbar";
+import PageTitle from "../PageTitle/PageTitle";
+import { updateBooking } from "../../parse-functions/updateFunctions";
+
+const EditBooking = () => {
+  const { bookingId } = useParams();
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState([]);
+
+  return (
+    <>
+      <NavBar />
+      <PageTitle ptitle="Edit booking"></PageTitle>
+      <form onSubmit={(e) => updateBooking(e, bookingId, formData, navigate)}>
+        <GrayContainer>
+          <GrayColumn>
+            <LabeledInput
```

Comment:
I think you use the same components as in the booking component. So instead of copy pasting we could make it reusable and pass the function, f.ex. updateBooking through props